assets/js/modules/analytics/datastore/settings.js

```diff
@@ -75,7 +75,7 @@ export const INVARIANT_INVALID_INTERNAL_PROPERTY_ID =
 	'cannot submit changes with incorrect internal webPropertyID';
 
 async function submitGA4Changes( { select, dispatch } ) {
-	if ( ! select( MODULES_ANALYTICS_4 ).haveSettingsChanged() ) {
+	if ( ! select( MODULES_ANALYTICS_4 ).haveAnyGA4SettingsChanged() ) {
```

Comment:
It's necessary to add these changes to the `analytics` `submitChanges()` and `validateCanSubmitChanges()` functions, as it's the `analytics` versions of `submitChanges()` / `canSubmitChanges()` that are invoked on the Settings page.